packages/legend-studio/src/components/editor/edit-panel/mapping-editor/MappingTestEditor.tsx

```diff
@@ -304,6 +312,21 @@ export const MappingTestFlatDataInputDataBuilder = observer(
   },
 );
 
+export const getRelationalInputTestDataEditorLanguage = (
+  type: RelationalInputType,
+): EDITOR_LANGUAGE => {
+  switch (type) {
+    case RelationalInputType.SQL:
+      return EDITOR_LANGUAGE.MARKDOWN;
```

Comment:
this looks weird, we have `EDITOR_LANGUAGE.SQL` I think

Reply:
Yes we do , made a mistake here, will be changing it